nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/main/java/org/nuxeo/elasticsearch/audit/ESAuditBackend.java

```diff
@@ -687,4 +687,49 @@ protected String getESIndexName() {
         return esa.getIndexNameForType(ElasticSearchConstants.ENTRY_TYPE);
     }
 
+    @Override
+    public long getLatestLogId(String repositoryId, String... eventId) {
+        long id = 0;
```

Comment:
Don't use this pattern of initializing a default value at the beginning. Just return the value when you have it, otherwise return a default value.